lib/cli/commands-schema/aws-service.js

```diff
@@ -26,7 +26,7 @@ commands.set('deploy', {
       type: 'boolean',
     },
     'function': {
-      usage: "Function name. Deploys a single function (see 'deploy function')",
+      usage: "Function name. Deploys a single function. DEPRECATED: use 'deploy function' instead.",
```

Comment:
According to proposal, it was suggested to remove this option from the schema - what was the reason against doing so?

Reply:
It needs to stay in schema, otherwise it will not be properly recognized and the expected deprecation won't be reported

Reply:
Yes, it's probably right to keep it (same way as keep schema for deprecated config properties)